README.md

```diff
@@ -165,6 +165,8 @@ Force fields moving forward will be called `name-X.Y.Z`
 
 - `2024.01.0`: This release adds `openff-2.1.1.offxml` and `openff_unconstrained-2.1.1.offxml`, Sage 2.1.1, which has identical parameters to Sage 2.1.0 (originally released in version `2023.05.1` of the `openff-forcefields` package) but adds Xe van der Waals parameters from [Tang, K.T., Toennies, J.P. New combining rules for well parameters and shapes of the van der Waals potential of mixed rare gas systems. Z Phys D - Atoms, Molecules and Clusters 1, 91–101 (1986).](https://doi.org/10.1007/BF01384663)
 
+- `2024.03.06`: This release adds `openff-2.2.0.offxml` and `openff_unconstrained-2.2.0.offxml`, Sage 2.2.0 Release Candidate 1, which modifies some small ring internal angles and remedies issues with sulfamide geometries.
```

Comment:
The versioning here is YYYY.MM.MINOR (We should add this to the README, I think it's copied over from another package.)

```suggestion
- `2024.03.0`: This release adds `openff-2.2.0-rc1.offxml` and `openff_unconstrained-2.2.0-rc1.offxml`, Sage 2.2.0 Release Candidate 1, which modifies some small ring internal angles and remedies issues with sulfamide geometries.
```

Is the plan to make a release (i.e. conda package) of this? I can coordinate this with @j-wags's signal

Reply:
Yes, the plan is to make a release+conda package with this. I'll do a quick review, then it'd be great if you coordinated the release.

Reply:
Just to double check--is changing it from `2024.03.06` --> `2024.03.0` the same change as changing it to YYYY.MM.MINOR ? I can change it to `2024.03.MINOR` if that's preferable, just let me know.